app/slo.py

```diff
@@ -4,13 +4,16 @@
 import fnmatch
 import logging
 import os
+import math
 
 import psycopg2
 import requests
 import zign.api
 
-logger = logging.getLogger('slo')
 KAIROS_QUERY_LIMIT = os.getenv('KAIROS_QUERY_LIMIT', 10000)
+MIN_VAL = math.expm1(1e-10)
+
+logger = logging.getLogger('slo')
```

Comment:
Kept it like that just to be consistent with other module loggers (not using the same name), but in the end they could all use the same logger.